yew-macro/src/html_classes/mod.rs

```diff
@@ -0,0 +1,32 @@
+use proc_macro2::{TokenStream};
+use quote::{quote, ToTokens};
+use syn::parse::{Parse, ParseStream, Result};
+use syn::punctuated::Punctuated;
+use syn::token::Comma;
+use syn::Expr;
+
+/// List of HTML classes.
+pub struct HtmlClasses(Punctuated::<Expr, Comma>);
+
+impl Parse for HtmlClasses {
+    fn parse(input: ParseStream) -> Result<Self> {
+        Ok(HtmlClasses(Punctuated::<Expr, Comma>::parse_terminated(input)?))
+    }
+}
+
+impl ToTokens for HtmlClasses {
+    fn to_tokens(&self, tokens: &mut TokenStream) {
+        let n = self.0.len();
+        let classes = self.0.iter();
+        let new_tokens = quote! {
+            let mut __yew_classes = ::yew::virtual_dom::Classes::with_capacity(#n);
+            #(__yew_classes.push(#classes);)*
+            __yew_classes
+        };
+
+        tokens.extend(quote! {{
+            #[allow(clippy::useless_conversion, unused_braces)]
+            #new_tokens
+        }});
+    }
+}
```

Comment:
The implementation of `classes!` is more trivial than the classes in `html!`. Here I have a comma separated list of expressions that I add (extend) one by on to `Classes`. If we remove the handling of classes in `html!`, the diff of this PR will be negative because the original implementation has more code.